lib/scribe.js

```diff
@@ -44,7 +44,7 @@ var Scribe = exports.Scribe = function(host, port, opt) {
     } else if (opt.autoReconnectTimeout.constructor === Array) {
       for (var i = 0; i < opt.autoReconnectTimeout.length; i++) {
         var timeout = opt.autoReconnectTimeout[i];
-        if (timeout.constructor === Number) {
+        if (timeout.constructor !== Number) {
```

Comment:
was that completely wrong before?

Reply:
not my fix because im just merging their master into ours but i would assume so since the error that otherwise would've been thrown is ` throw new Error("autoReconnectTimeout should be Number or Array of numbers");` :S

Reply:
Yeah looks like a bug before that they fixed. The message is a little confusing but correct, they are already going through the array just making sure each one is a number.